lib/block.ml

```diff
@@ -324,6 +330,17 @@ let guess_ocaml_kind b =
 let toplevel ~file ~line lines = Toplevel (Toplevel.of_lines ~line ~file lines)
 
 let eval t =
+  let re_tab = Re.(compile (char '\t')) in
+  let on_line i s =
+    let s' =  Re.replace_string re_tab ~by:"    " s in
```

Comment:
This is replacing all the tabs and we only need to replace the leading tabs. Also, tabs that are not at the beginning of the line doesn't always have the same size.
Maybe we don't need the full power of `Re` here. It may also make things more complicated here.